tests/run.sh

```diff
@@ -9,5 +9,17 @@ set -e
 # - SPARK_VERSION
 # Default values can be looked up in tests/utils/common.go
 pushd $TEST_DIR
-go test -v -count=1 -timeout 30m .
+
+if [ -n "$TEAMCITY_VERSION" ]; then
+  # Running in Teamcity, therefore we need to convert `go test` output accordingly
+  if command -v go-test-teamcity; then
+    go test -v -count=1 -timeout 30m . | go-test-teamcity
```

Comment:
`go test -v -count=1 -timeout 30m` repeated three times here, shall we extract it to a variable?

also, it would be great to make `timeout` configurable and allow overriding it via an environment variable.